### PR TITLE
reducer testing 

### PR DESCRIPTION
**Pull request by hanyulo**
author-articles, authors reducer testing are done!!

### CODE REVIEW

File: src/reducers/__test__/authors.test.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    it('should return expected state when previous state is initial state', function () {
+      const initialState = mockStates.InitialState
+      const expectedState = { ...mockStates.InitialState, isFetching: true }
+      expect(
+        reducer(initialState, mockActions.SEARCH_AUTHORS_REQUEST)
+      ).to.deep.equal(expectedState)
+    })
+    it('should return the expected state when previous state exist', function () {
+      const preState = mockStates.ExpStateSucwithInit
+      const expectedState = { ...mockStates.ExpStateSucwithInit, isFetching: true }
```
Comment (nickhsine) [inline src/reducers/__test__/authors.test.js]: no `keywords` in `types.SEARCH_AUTHORS_REQUEST` action ?

**Comment by nickhsine** [comment]
Could this PR be reviewed now?

**Comment by hanyulo** [comment]
Yes! This one is ready now. Thx

**Comment by nickhsine** [comment]
💯 👍

**Pull request merged**